### PR TITLE
DOC-580 | Improve Docs styling and add ArangoGraph Banner

### PR DESCRIPTION
**Pull request by nikhil-varma**
### Description
- This PR improves the overall styling of the docs and matches the ArangoGraph's UI styles.
- Also adds ArangoGraph's banner to the home page of each Doc version specifically 3.10, 3.11 & 3.12

### CODE REVIEW

File: site/themes/arangodb-docs-theme/static/css/theme-relearn-light.css [hunk 1]
```diff
@@ -108,24 +108,24 @@
...
-    --TIP-BG-COLOR: var(--green-50);
+    --TIP-BG-COLOR: transparent;
     --TIP-BORDER-COLOR: var(--green-600);
 
-    --SECURITY-ICON-COLOR: #FF9800;
-    --SECURITY-BG-COLOR: #FFF3E0;
-    --SECURITY-BORDER-COLOR: #FF9800;
+    --SECURITY-ICON-COLOR: #FFA500;
+    --SECURITY-BG-COLOR: transparent;
+    --SECURITY-BORDER-COLOR: #FFA500;
```
Comment (palashkaria) [inline site/themes/arangodb-docs-theme/static/css/theme-relearn-light.css]: Let's use the var(--orange-400) here

File: site/themes/arangodb-docs-theme/static/css/theme-relearn-light.css [hunk 2]
```diff
@@ -108,24 +108,24 @@
     --TOC-ENTRY-COLOR: #4a4a4a;
 
     /* ADMONITIONS */
-    --INFO-ICON-COLOR: var(--blue-500);
-    --INFO-BG-COLOR: var(--blue-100);
-    --INFO-BORDER-COLOR: var(--blue-500);
+    --INFO-ICON-COLOR: #276f86; /* var(--blue-500);*/
```
Comment (palashkaria) [inline site/themes/arangodb-docs-theme/static/css/theme-relearn-light.css]: let's keep using the blue variable we have

File: site/themes/arangodb-docs-theme/static/css/theme-relearn-light.css [hunk 3]
```diff
@@ -108,24 +108,24 @@
...
 
     --WARNING-ICON-COLOR: var(--orange-400);
     --WARNING-BORDER-COLOR: var(--orange-400);
-    --WARNING-BG-COLOR: var(--orange-100);
+    --WARNING-BG-COLOR: transparent;
 
-    --DANGER-ICON-COLOR: var(--red-500);
-    --DANGER-BORDER-COLOR: var(--red-500);
-    --DANGER-BG-COLOR: var(--red-50);
+    --DANGER-ICON-COLOR: #fa8151;
```
Comment (palashkaria) [inline site/themes/arangodb-docs-theme/static/css/theme-relearn-light.css]: we can keep using the red here

File: site/themes/arangodb-docs-theme/static/css/theme.css [hunk 4]
```diff
@@ -1026,14 +1043,19 @@ header .logo {
 }
 
 #breadcrumbs > ol > li > a {
-    font-weight: 700;
-    color: var(--green-700);
-    font-size: var(--TYPOGRAPHY-BODY-TEXT);
+    font-size: 1rem;
+    font-weight: 500;
+    color: var(--gray-700);
+    border-bottom: 0;;
```
Comment (palashkaria) [inline site/themes/arangodb-docs-theme/static/css/theme.css:1049]:
```suggestion
    border-bottom: 0;
```

File: site/themes/arangodb-docs-theme/static/css/theme.css [hunk 5]
```diff
@@ -1371,40 +1395,48 @@ h6:hover .header-link {
 
 
 /* Codeblocks */
-pre code {
-    color: rgb(61 63 66);
+pre {
+    border-radius: 0px;
+    background: rgb(240, 240, 240);
```
Comment (palashkaria) [inline site/themes/arangodb-docs-theme/static/css/theme.css:1400]: this is gray-100

File: site/themes/arangodb-docs-theme/static/css/theme.css [hunk 6]
```diff
@@ -1371,40 +1395,48 @@ h6:hover .header-link {
...
     font-size: 14px;
     overflow: auto;
     word-wrap: break-word;
-    background-color: var(--CODE-BG-COLOR);
     margin: 0;
     padding: 2px;
     word-break: break-word;
-    color: var(--CODE-COLOR);
+    border-radius: 0px;
+    background: rgb(240, 240, 240);
```
Comment (palashkaria) [inline site/themes/arangodb-docs-theme/static/css/theme.css:1413]: gray-100

File: site/themes/arangodb-docs-theme/static/css/theme.css [hunk 7]
```diff
@@ -1371,40 +1395,48 @@ h6:hover .header-link {
...
+    box-shadow: none;
+    color: rgb(68, 68, 68);
```
Comment (palashkaria) [inline site/themes/arangodb-docs-theme/static/css/theme.css:1415]: gray-900

File: site/themes/arangodb-docs-theme/static/css/theme.css [hunk 8]
```diff
@@ -1371,40 +1395,48 @@ h6:hover .header-link {
...
-    height: 40px;
-    background: var(--gray-200);
     display: flex;
     align-items: center;
     justify-content: center;
     position: absolute;
     right: 0;
-    border-radius: 20px;
     cursor: pointer;
+    background: #e3e3e3;
```
Comment (palashkaria) [inline site/themes/arangodb-docs-theme/static/css/theme.css:1431]: gray-200

File: site/themes/arangodb-docs-theme/static/css/theme.css [hunk 9]
```diff
@@ -1421,14 +1453,26 @@ li > code {
...
+    padding: 0 5px;
+    font-weight: bold;
+    color: #53785c;
+}
+.copy-to-clipboard-button.tooltipped::after{
+    content: 'Copied';
+    font-family: 'Inter', sans-serif;
+    padding: 0 5px;
+    font-weight: bold;
+    color: #53785c;
```
Comment (palashkaria) [inline site/themes/arangodb-docs-theme/static/css/theme.css:1468]: green-700

File: site/themes/arangodb-docs-theme/static/css/theme.css [hunk 10]
```diff
@@ -1443,14 +1487,40 @@ li > code {
 
 pre > .code-show-more {
     font-size: 1rem;
-    box-shadow: 0px 0px 0px #1c1616, 0px -2px 1px #888;
-    margin: 0;
-    border-bottom-left-radius: 24px;
-    border-bottom-right-radius: 24px;
+    background: #e3e3e3;
```
Comment (palashkaria) [inline site/themes/arangodb-docs-theme/static/css/theme.css:1490]: gray-200

**Review by palashkaria — CHANGES_REQUESTED** [review]

File: site/themes/arangodb-docs-theme/static/css/theme-relearn-light.css [hunk 11]
```diff
@@ -108,24 +108,23 @@
     --TOC-ENTRY-COLOR: #4a4a4a;
 
     /* ADMONITIONS */
-    --INFO-ICON-COLOR: var(--blue-500);
-    --INFO-BG-COLOR: var(--blue-100);
+    --INFO-ICON-COLOR:  var(--blue-500);
```
Comment (palashkaria) [inline site/themes/arangodb-docs-theme/static/css/theme-relearn-light.css:111]:
```suggestion
    --INFO-ICON-COLOR: var(--blue-500);
```

File: site/themes/arangodb-docs-theme/static/css/theme.css [hunk 12]
```diff
@@ -1371,40 +1395,48 @@ h6:hover .header-link {
...
     display: flex;
     align-items: center;
     justify-content: center;
     position: absolute;
     right: 0;
-    border-radius: 20px;
     cursor: pointer;
+    background: var(--gray-200);
+    padding: 3px 10px;
+    margin: 7px;
```
Comment (palashkaria) [inline site/themes/arangodb-docs-theme/static/css/theme.css:1433]: Lets try to stick to multiples of 4/2

**Comment by Simran-B** [comment]
The text contrast of inline code is not good, and way too low if it's in a link. The link underline is also not visible below inline code (see "`db` object" link):

image

It doesn't look good and has too little contrast in headlines as well:
image

The copy button overlapping the content is not nice. There would be space to make the code box wider in this case, but to improve it more generally, could we show the copy button only on hover over the code box?

image

Boxes like [here](https://deploy-preview-244--docs-hugo.netlify.app/3.11/develop/http/collections/#get-the-properties-of-a-collection_res_200_cacheEnabled) need to be restyled still. Having the same border radius on nested boxes looks wrong:

image

The margin between list items for dense lists seems to be the same as the regular line height.

**Comment by nikhil-varma** [comment]
@Simran-B 
Links & contrast are fixed already. Will get the inline code sorted.

Any further changes would be a part of a separate ticket. It becomes tricky to handle details. Let's keep the scope as-is for this one. Please use this ticket to write down whatever is necessary for the next set of issues: https://arangodb.atlassian.net/browse/DOC-585

**Review by palashkaria — APPROVED** [review]
LGTM!

**Pull request merged**